src/datasets/info.py

```diff
@@ -282,6 +283,12 @@ def from_merge(cls, dataset_infos: List["DatasetInfo"]):
         supervised_keys = None
         task_templates = None
 
+        repo_ids = {dset_info.repo_id for dset_info in dataset_infos}
+        if len(repo_ids) == 1:
+            repo_id = next(iter(repo_ids))
```

Comment:
```suggestion
            repo_id = repo_ids.pop()
```
This is usually my goto here